index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css"/>
+
+	<title>My Portifolio Page</title>
+	<link rel="stylesheet" type="text/css" href="styles.css">
+</head>
+<body>
```

Comment:
Your application at this point is missing the background image as in the Figma design. Consider adding it to make it look like the provided design template.